eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -307,7 +310,11 @@ def emceefitter(lc, model, meta, log, **kwargs):
         if len(ind)>1:
             for chan in np.arange(lc.flux.size//lc.time.size):
                 lc.unc_fit[chan*lc.time.size:(chan+1)*lc.time.size] = fit_params[ind[chan]] * 1e-6
-
+        
+    #Plot GP fit + components
+    if model.GP:
+        plots.plot_GP_components(lc, model, meta, fitter='emcee')
```

Comment:
Is there a reason the lsqfitter doesn't get a similar plot?

Reply:
Problem is that the lsq fit doesn't seem to be fitting the GP and I'm not entirely sure why, that's the reason there's no GP plot for that yet. I'll raise it as an issue and assign myself to look into that after this.